composer.json

```diff
@@ -50,15 +50,14 @@
         "doctrine/dbal": "2.13.0 || 2.13.1"
     },
     "require-dev": {
-        "typo3/cms-reports": "~11.4.0 || dev-master",
-        "typo3/cms-filemetadata": "~11.4.0 || dev-master",
+        "typo3/cms-reports": "^11.4.0 || dev-master",
+        "typo3/cms-filemetadata": "^11.4.0 || dev-master",
         "typo3-console/create-reference-command": "@dev",
         "symfony/expression-language": "^4.4 || ^5.0",
         "symfony/filesystem": "^4.4 || ^5.0",
         "nimut/testing-framework": "dev-php8",
         "phpunit/phpunit": "^8.5.18",
-        "php-parallel-lint/php-parallel-lint": "^1.2",
-        "typo3-console/php-server-command": "@dev"
```

Comment:
Is there a reason why `typo3-console/php-server-command` was dropped?

Reply:
yes

Reply:
Can you explain/mention it? ;-)

Reply:
It was never really needed for anything and it again has issues with not being able to be installed: https://github.com/TYPO3-Console/TYPO3-Console/runs/3753980412?check_suite_focus=true

Reply:
I see, thanks for the insights. :+1: